lib/samples/display_clusters/display_clusters_sample.dart

```diff
@@ -25,28 +25,31 @@ class DisplayClustersSample extends StatefulWidget {
 }
 
 class _DisplayClustersSampleState extends State<DisplayClustersSample> {
+  // create a map view controller
   final _mapViewController = ArcGISMapView.createController();
-  final _map = ArcGISMap.withUri(
-    Uri.parse(
-        'https://www.arcgis.com/home/item.html?id=8916d50c44c746c1aafae001552bad23'),
-  )!;
-
+  // creates a map
```

Comment:
```suggestion
```
Maybe delete this comment as we configure the map later and have a comment with more detail.